lib/rack_middleware/enforce_ssl.rb

```diff
@@ -0,0 +1,21 @@
+class EnforceSsl
```

Comment:
Yes. I considered as stated in the `application.rb`. 

```
 # Manually redirect http to https, if config option always_use_ssl is set to true
 # This needs to be done before routing: conditional routes break if this is done later
 # Enabling HSTS and secure cookies is not a possiblity because of potential reuse of domains without HTTPS